apps/core/tests/test_environment_config_view.py

```
@@ -0,0 +1,72 @@
+        environment,
+        "_get_django_settings",
+        lambda: [
+            ("AWS_ACCESS_KEY_ID", "key"),
+            ("AWS_SECRET_ACCESS_KEY", "secret"),
+            ("DEBUG", True),
+        ],
+    )
+    user = get_user_model().objects.create_superuser(
+        username="admin",
+        email="admin@example.com",
+        password="admin123",
```

Ruff (0.15.1)

[error] Possible hardcoded password assigned to argument: "password"

(S106)